src/themes/playful.css

```
@@ -244,22 +260,22 @@
   /* Border widths are designed to scale according to a single base value.
      * The base value is intended for calculations and is not used by components directly. */
   --wa-border-width-base: 0.125;
-  --wa-border-width-s: calc(var(--wa-border-width-base) * 1rem);
+  --wa-border-width-s: calc(var(--wa-border-width-base) * 1.5rem);
   --wa-border-width-m: calc(var(--wa-border-width-base) * 2rem);
   --wa-border-width-l: calc(var(--wa-border-width-base) * 3rem);
 
   --wa-form-controls-border-style: var(--wa-border-style);
-  --wa-form-controls-border-width: var(--wa-border-width-s);
+  --wa-form-controls-border-width: var(--wa-border-width-m);
 
   --wa-panel-border-style: var(--wa-border-style);
-  --wa-panel-border-width: var(--wa-border-width-s);
+  --wa-panel-border-width: 0;
 
   /**
    * Corners
    */
   /* Corners are designed to scale according to a single base value.
    * The base value is intended for calculations and is not used by components directly. */
-  --wa-corners-base: 0.5;
+  --wa-corners-base: 0.75;
```

```suggestion
  --wa-corners-base: calc(8 / 18);
```

This one's a bit trickier, but we could use a similar `calc()` approach here. Looks like we have 8px, 12px, 24px, and 40px corners at 100% zoom throughout this theme, so 8 might be a good starting point for our base. We can set that in our base and adjust the calculations below multiply by `1rem`, `1.5rem`, `3rem`, and `5rem`.